src/Infrastructure.EntityFramework/Models/Cache.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Infrastructure.EntityFramework.Models;
+
+public class Cache
+{
+    [StringLength(499)]
```

Comment:
:x: Different value than the table definition.

Reply:
Matches the SqlServer table definition below.

Reply:
One is 449, other is 499.

Reply:
Totally missed that!